src/lib/fetcher.js

```diff
@@ -0,0 +1,28 @@
+// we use JS here instead of TS because of an ongoing issue in @graphiql/toolkit where
+// graphql-ws needs to be updated. Types don't match currently and will be addressed when
+// https://github.com/graphql/graphiql/pull/2017 is released. For now, we use JS to
```

Comment:
Isn't it released already ? https://github.com/graphql/graphiql/commit/a3782ff0ff0d7c321e6f70bea61b1969b1690f26

Reply:
Yes. Will make the required change.